### PR TITLE
[TIMOB-17450] TiAPI: Ti.Calendar.Event should expose the attendees of the meeting invite as a property

### PR DESCRIPTION
**Pull request by sriks**
JIRA: https://jira.appcelerator.org/browse/TIMOB-17450

### CODE REVIEW

**Comment by ashcoding** [comment]
Is this ready for review?

**Comment by sriks** [comment]
@ashcoding Not yet. Im due to commit changes to iOS.

**Comment by ashcoding** [comment]
Roger that.

**Comment by ashcoding** [comment]
Thanks for the update!

File: android/modules/calendar/src/java/ti/modules/titanium/calendar/AttendeeProxy.java [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    public AttendeeProxy() {
+        super();
+    }
+
+    public AttendeeProxy(TiContext context) {
+        this();
+    }
+
+    public static ArrayList<AttendeeProxy> fetchAttendees(final long eventId, ContentResolver contentResolver) {
+        ArrayList<AttendeeProxy> attendees = new ArrayList<>();
```
Comment (ashcoding) [inline android/modules/calendar/src/java/ti/modules/titanium/calendar/AttendeeProxy.java]: Looks like travis is failing here.

```
/Users/travis/build/appcelerator/titanium_mobile/android/modules/calendar/src/java/ti/modules/titanium/calendar/AttendeeProxy.java:53: error: diamond operator is not supported in -source 1.6 [javac] ArrayList<AttendeeProxy> attendees = new ArrayList<>(); [javac] ^ [javac] (use -source 7 or higher to enable diamond operator)
```
Could 
```
ArrayList<AttendeeProxy> attendees = new ArrayList<AttendeeProxy>();
```
Resolve this?

**Comment by ashcoding** [comment]
Will see if https://github.com/appcelerator/titanium_mobile/pull/8049 solves this.

**Comment by ashcoding** [comment]
Just a note, can't compile it due to the usage of diamond operator, which is not supported.
Will be closing this PR for now and will require a new PR for a fix for Android separately.

@hansemannn if the iOS is okay, you can probably cherry-pick them out as a separate PR.

**Comment by mrkiley** [comment]
@ashcoding / @sgtcoolguy: Did you guys have a different PR that you opened for this for Android yet? Sorry, but I can't seem to find it if so.

**Comment by ashcoding** [comment]
There's no different PR for this. This was rejected as it can't be compiled.

**Pull request closed without merging**